op-challenger/config/config.go

```diff
@@ -32,10 +32,30 @@ type TraceType string
 const (
 	TraceTypeAlphabet TraceType = "alphabet"
 	TraceTypeCannon   TraceType = "cannon"
+
+	// Devnet game IDs
+	DevnetGameIDAlphabet = uint8(0)
+	DevnetGameIDCannon   = uint8(1)
+
+	// Mainnet game IDs
+	MainnetGameIDFault = uint8(0)
 )
 
 var TraceTypes = []TraceType{TraceTypeAlphabet, TraceTypeCannon}
 
+// GameIdToString maps game IDs to their string representation on a per-network basis.
+var GameIdToString = map[uint64]map[uint8]string{
```

Comment:
I didn't realize that we these IDs meant different things based on the network - I know it's weird to put in the alphabet game on mainnet, but it'd save a bit of complexity in configuration.

Reply:
Yep - It's a bit odd, but would be nice if we can support different game type mappings for the various networks (we only have 255 available types, so best not to pollute mainnet's imo). Glad to think of a way to standardize them across the networks if we want this, though.

Reply:
One temporary solution is to deploy devnet-specific games with game types descending from 255, and it'll probably be a long time (if ever) before we'd need to worry about them colliding. Still would be more game types on devnet than mainnet, but the game types that exist on mainnet would have the same values on devnet.

Reply:
yea, that's not a bad solution. Let's merge this PR as is, but I do want to follow up on this.